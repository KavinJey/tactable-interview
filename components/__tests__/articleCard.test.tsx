import ArticleCard from "../articleCard";
import { render, screen } from '@testing-library/react'

describe('Article Card', () => {
  it('renders a heading', () => {
    render(<ArticleCard />)

    const heading = screen.getByRole('heading', {
      name: /welcome to next\.js!/i,
    })

    expect(heading).toBeInTheDocument()
  })

  it('renders homepage unchanged', () => {
  const { container } = render(<ArticleCard />)
  expect(container).toMatchSnapshot()
})

  
})