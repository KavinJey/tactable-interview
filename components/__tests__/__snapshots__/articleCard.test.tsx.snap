// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article Card renders homepage unchanged 1`] = `
<div>
  <div />
</div>
`;
